docs/pages/components/tree.md

```diff
@@ -8,447 +8,1867 @@ permalink: components/tree.html
-            Column Header
-        </div>
-        <div class="fd-tree__col">
-            Status
-        </div>
-        <div class="fd-tree__col fd-tree__col--actions">
-        </div>
-    </div>
-</div>
-<ul class="fd-tree" id="tWsod582" role="tree">
-    <li class="fd-tree__item" role="treeitem" id="inYUX852" aria-expanded="false">
-        <div class="fd-tree__row">
-            <div class="fd-tree__col fd-tree__col--control">
-                <button class="fd-tree__control" aria-label="Expand" aria-controls="inYUX852"
-                aria-pressed="false"></button>
-                First Level
-            </div>
-            <div class="fd-tree__col">
-                Secondary Value
-            </div>
-            <div class="fd-tree__col">
-                Secondary Value
-            </div>
-            <div class="fd-tree__col">
-                    Inactive
-            </div>
-            <div class="fd-tree__col fd-tree__col--actions">
-               <div class="fd-popover fd-popover--right">
-                    <div class="fd-popover__control">
-                        <button class="fd-button fd-button--transparent sap-icon--overflow" aria-controls="j2lk3j" aria-haspopup="true" aria-expanded="false" aria-label="More"></button>
-                    </div>
-                    <div class="fd-popover__body fd-popover__body--right" aria-hidden="true" id="j2lk3j">
-                        <nav class="fd-menu">
-                            <ul class="fd-menu__list fd-menu__list--no-shadow">
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Edit</span>
-                                    </a>
-                                </li>
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Lock</span>
-                                    </a>
-                                </li>
-                                <li class="fd-menu__item">
-                                    <a href="#" class="fd-menu__link">
-                                        <span class="fd-menu__title">Duplicate</span>
-                                    </a>
+<ul role="tree" id="TREE1L1" class="fd-tree level-1 expanded-level-1">
+    <li role="treeitem" aria-level="1" aria-expanded="true" class="fd-tree__item">
+        <div tabindex="0" class="fd-tree__item-container">
+            <span role="presentation" class="fd-tree__expander is-expanded" aria-controls="TREE1L2" aria-haspopup="true"  aria-label="Expand level 2"></span>
+            <div class="fd-tree__content">
+                <span class="fd-tree__text">Level 1</span>
+            </div>
+        </div>
+        <ul role="tree" class="fd-tree level-2" id="TREE1L2" aria-hidden="false">
```

Comment:
Only the first `ul` should have a role of `tree`. Subsequent lists should have the role of `group`. See this markup example: http://oaa-accessibility.org/examplep/treeview1/

There should also be a label for the entire tree, either a visible `label` node or an aria-label on the `tree`.

The trigger to expand the subtree should be a `button` instead of a `span`.

Reply:
Hi @meganaconley Thank you for reviewing the PR. All your comments have been addressed.